src/main/java/core/basesyntax/service/impl/operation/ReturnOperation.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl.operation;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.Fruit;
+
+public class ReturnOperation implements OperationHandler {
+    private final FruitStorageDao fruitStorageDao;
+
+    public ReturnOperation(FruitStorageDao fruitStorageDao) {
+        this.fruitStorageDao = fruitStorageDao;
+    }
+
+    @Override
+    public boolean doOperation(Fruit fruit, int quantity) {
+        if (fruit == null) {
+            throw new RuntimeException("Fruit can't be null");
+        }
+        if (quantity < MIN_FRUIT_QUANTITY) {
+            throw new RuntimeException("Quantity " + quantity + " can't be negative");
+        }
```

Comment:
According to the checklist, handling the 'Return' operation should not be different from 'Supply' operation as they both increase the amount of fruit in the storage. Consider reusing the logic from 'SupplyOperation' if it's identical, adhering to the DRY principle.